website/templates/emails/reviews_submission_status.html.mako

```diff
@@ -60,7 +60,7 @@
 
                                 <li>Upload supplemental, materials, data, and code to the OSF project associated with your ${document_type}.
                                     <a href="https://help.osf.io/article/177-upload-a-preprint" target="_blank">Learn how</a></li>
-                                <li>Preregister your next study. <a href="https://help.osf.io/345-create-registrations">Read more</a></li>
+                                <li>Preregister your next study. See <a href="https://help.osf.io/345-create-registrations">Create a Registration</a></li>
```

Comment:
```suggestion
                                <li>Preregister your next study. Learn how to <a href="https://help.osf.io/345-create-registrations">create a registration</a></li>
```

Sorry, missed this in the initial review as well.

Reply:
I will note that it's a little odd that it doesn't act more like a sentence (note that the `Learn how` above is similar) with a trailing `.`.